nvme_dimm_fix.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -uex
+
+tnode=${NODELIST##*,}
+
+idstr='Non-Volatile memory controller: '
+idstr+='Intel Corporation NVMe Datacenter SSD.*Optane'
+nvme_ids=($(ssh ${SSH_KEY_ARGS} root@${tnode} \
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]